File: Service/Implement/DanhMucCompanyInfoService.cs
namespace Service.Implement
{
    public class DanhMucCompanyInfoService : BaseService<DanhMucCompanyInfo, IDanhMucCompanyInfoRepository>
    , IDanhMucCompanyInfoService
    {
    private readonly IDanhMucCompanyInfoRepository _DanhMucCompanyInfoRepository;
    public DanhMucCompanyInfoService(IDanhMucCompanyInfoRepository DanhMucCompanyInfoRepository) : base(DanhMucCompanyInfoRepository)
    {
    _DanhMucCompanyInfoRepository = DanhMucCompanyInfoRepository;
    }
    }
    }

File: CMS/src/app/shared/DanhMucCompanyInfo.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucCompanyInfo } from 'src/app/shared/DanhMucCompanyInfo.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucCompanyInfoService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucCompanyInfo";
    }
}

